samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,274 @@
+# CC-Tools-Demo Tutorial
+
+This tutorial shows how to build, install and test a Go Chaincode developed using the [CC-Tools]() framework and integrating it with the Fabric Private Chaincode (FPC) framework.
+
+This tutorial illustrates a simple use case where we follow the [cc-tools-demo]() chaincode which is based on standard fabric and then convert it to an FPC chaincode achieving FPC security capabilities.
+
+This tutorial is based on the [FPC with CC-Tools integration project]() and all our design choices are explained here in the [design document]()
+Here are the steps to accomplish this:
+
+* Clone and copy the cc-tools-demo chaincode
```

Comment:
```suggestion
* Clone the cc-tools-demo chaincode
```

Reply:
But I don't need the whole cc-tools-demo repo, I only want the chaincode as the repo includes multiple other files

Reply:
Done